harmony_netcdf_to_zarr/rechunk.py

```diff
@@ -123,3 +125,19 @@ def is_group(name: str) -> None:
     original_zarr.visit(is_group)
 
     return groups
+
+
+def _copy_group_attributes(source_loc, target_loc):
+    """Visit every source group and copy any attributes to the corresponding target group."""
```

Comment:
Is it worth updating this documentation string to state why this is being done and/or that we hope to remove this when the rechunker does it natively?

Reply:
Yes, I can add a link to the issue and mention to remove when the issue is closed and we update rechunker lib